client/src/utils/helpers.tsx

```diff
@@ -177,6 +177,193 @@ const filterUsersByTeams = (teams: string[], directory: IUser[]): IUser[] => {
   return filteredDirectory;
 };
 
+/**
+ * Gets all of the teams associated with a pitch
+ *
+ * A pitch has a team when the {TEAM}.target > 0
+ *
+ * @param pitch the pitch to pull the teams from
+ * @returns an array of all of teams belonging to the pitch
+ */
+const getPitchTeams = (pitch: IPitch): string[] => {
+  const teams: string[] = [];
+
+  const pitchTeams = Object.entries(pitch.teams);
+
+  pitchTeams.forEach((team) => {
+    const teamName: string = team[0];
+    const assingments = team[1];
+
+    if (assingments.target > 0) {
```

Comment:
Ok 2 things here:

1. You spelled `assignments` wrong
2. Instead of checking if `assignments.target > 0`, you check if `current >= target` for each team